dedibox-network/rpn/concepts.mdx

```diff
@@ -62,7 +62,7 @@ RPN-v2 is a new version of Dedibox RPN Network (Real Private Network). It allows
 ## RPN VPN
 
 RPN VPN allows you to connect to your RPN group using a VPN. The VPN is based on OpenVPN to ensure easy configuration. You can add the VPN server to your RPN group as any other server.
-Notably, it allows you to securely connect from anywhere in your private network as well as to connect servers which don't have a RPN interface to the RPN.
+Notably, it allows you to securely connect from anywhere in your Private Network as well as to connect servers which don't have a RPN interface to the RPN.
```

Comment:
I think here it isn't a product name (only for Elements)

Reply:
```suggestion
Notably, it allows you to securely connect from anywhere in your private network as well as to connect servers that do not have a RPN interface to the RPN.
```